pynicotine/gtkgui/search.py

```diff
@@ -918,33 +909,35 @@ def check_filter(self, row):
         if self.active_filter_count == 0:
             return True
 
-        filters = self.filters
+        file_path = row[11].lower()
```

Comment:
`.lower()` should be done under the two conditions that require it in (`"filterin"`, `"filterout"`), because most of the filter_id's don't use the (potentially quite long) string value in this variable, and much of the time it is most likely (the default) that no filters.items() will be set anyway.